tests/30rooms/01state.pl

```diff
@@ -242,14 +242,24 @@ sub get_test_state
 {
    my ( $user, $room_id, $type ) = @_;
 
-   matrix_initialsync_room( $user, $room_id, limit => 0 )->then( sub {
-      my ( $body ) = @_;
+   matrix_sync( $user,
+      filter            => '{"room":{"rooms":["'.$room_id.'"]},"account_data":{},"presence":{"types":[]}}',
+      update_next_batch => 0,
+      set_presence      => "offline",
+   )->then( sub {
+        my ( $body ) = @_;
```

Comment:
I'm no fan of the 3-space indentation convention, but 5 spaces doesn't seem an improvement...

Reply:
VS Code yet again :/